velox/exec/RowNumber.cpp

```diff
@@ -64,6 +70,13 @@ void RowNumber::addInput(RowVectorPtr input) {
   const auto numInput = input->size();
 
   if (table_) {
+    ensureInputFits(input);
+
```

Comment:
Put NonReclaimableSection here to prevent memory reclaim from memory arbitrator?